### PR TITLE
Adjust the module to work with Fybrik 0.6.0.

### LINKED ISSUE #23: Support helm 3.7
In helm release 3.7 the commands related to OCI registry support have changed. More information about the change can be found in https://github.com/helm/community/blob/main/hips/hip-0006.md.
This issue is about supporting the new commands related to OCI registry in helm 3.7.

### LINKED ISSUE #30: Update module resources to Fybrik 0.6.0

### PR DESCRIPTION
**Pull request by revit13**
This PR updates the module to work with Fybrik 0.6.0, it contains:
- use of Helm 3.7 to deploy the module helm chart in OCI registry
- update to Fybrik resources used in the example in the README 
- updates to the README file.
Closes #23 
Closes #30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +102,7 @@ Here is an example how to deploy and test the module on a single cluster.
 
 ### Before you begin
 
-Install Fybrik using the [Quick Start](https://fybrik.io/v0.5/get-started/quickstart/) guide. This sample assumes the use of the built-in catalog, Open Policy Agent (OPA) and flight module.
+Install Fybrik using the [Quick Start](https://fybrik.io/dev/get-started/quickstart/) guide. This sample assumes the use of the built-in catalog, Open Policy Agent (OPA) and flight module.
```
Comment (Mohammad-nassar10) [inline README.md:105]: Do we need the `dev` version of the quick start or the `v0.6`?

**Review by Mohammad-nassar10 — APPROVED** [review]

File: hello-world-read-module.yaml [hunk 2]
```diff
@@ -16,22 +16,21 @@ spec:
...
         - source:
             protocol: s3
             dataformat: parquet
         - source:
             protocol: s3
             dataformat: csv
+      actions:
+        - name: RedactAction
+        - name: RemoveAction
   statusIndicators:
```
Comment (shlomitk1) [inline hello-world-read-module.yaml]: Remove status indicators. No custom resources exist.

File: README.md (hunk 1 above)
Comment (revit13) [reply README.md:105]: I think dev. The user is referred to the version compatibility matrix to choose the appropriate version of Fybrik....

File: hello-world-read-module.yaml (hunk 2 above)
Comment (revit13) [reply hello-world-read-module.yaml]: Done. Thanks

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**